test/e2e/pods.go

```diff
@@ -555,6 +578,233 @@ var _ = Describe("Pods", func() {
+		defer func() {
+			By("deleting the pod")
+			podClient.Delete(pod.Name, api.NewDeleteOptions(0))
+		}()
+		pod, err := podClient.Create(pod)
+		if err != nil {
+			Failf("Failed to create pod: %v", err)
+		}
+
+		expectNoError(framework.WaitForPodRunning(pod.Name))
+
+		By("verifying the pod is in kubernetes")
+		pod, err = podClient.Get(name)
+		if err != nil {
+			Failf("failed to get pod: %v", err)
+		}
+
+		By("geting restart delay-1")
+		delay1, err := getRestartDelay(framework.Client, pod, framework.Namespace.Name, name, containerName)
+		if err != nil {
+			Failf("timed out waiting for container restart in pod=%s/%s", name, containerName)
+		}
+
+		By("getting restart delay-2")
+		delay2, err := getRestartDelay(framework.Client, pod, framework.Namespace.Name, name, containerName)
+		if err != nil {
+			Failf("timed out waiting for container restart in pod=%s/%s", name, containerName)
+		}
+
+		ratio := float64(delay2) / float64(delay1)
+		if math.Floor(ratio) != 1 && math.Ceil(ratio) != 1 {
+			Failf("back-off increasing on LivenessProbe failure delay1=%s delay2=%s", delay1, delay2)
+		}
+	})
+
+	It("[Skipped] should cap back-off at 5mins", func() {
+		podClient := framework.Client.Pods(framework.Namespace.Name)
+		name := "back-off-cap"
+		containerName := "back-off-cap"
+		pod := &api.Pod{
+			ObjectMeta: api.ObjectMeta{
+				Name:   name,
+				Labels: map[string]string{"test": "liveness"},
+			},
+			Spec: api.PodSpec{
+				Containers: []api.Container{
+					{
+						Name:    containerName,
+						Image:   "gcr.io/google_containers/busybox",
+						Command: []string{"/bin/sh", "-c", "sleep 1", "/crash/missing"},
+						LivenessProbe: &api.Probe{
+							Handler: api.Handler{
+								Exec: &api.ExecAction{
+									Command: []string{"cat", "/tmp/health"},
+								},
+							},
+							InitialDelaySeconds: 2,
+						},
+					},
+				},
+			},
+		}
+
+		By("submitting the pod to kubernetes")
+		defer func() {
+			By("deleting the pod")
+			podClient.Delete(pod.Name, api.NewDeleteOptions(0))
+		}()
+		pod, err := podClient.Create(pod)
+		if err != nil {
+			Failf("Failed to create pod: %v", err)
+		}
+
+		expectNoError(framework.WaitForPodRunning(pod.Name))
+
+		By("verifying the pod is in kubernetes")
+		pod, err = podClient.Get(name)
+		if err != nil {
+			Failf("failed to get pod: %v", err)
+		}
+
+		time.Sleep(10 * time.Minute) // build to max back-off @ 1s sync interval
+
+		// wait for a delay == capped delay of 5mins
+		By("geting restart delay when capped")
+		var delay1 time.Duration
+		for i := 0; i < 3; i++ {
+			delay1, err = getRestartDelay(framework.Client, pod, framework.Namespace.Name, name, containerName)
+			if err != nil {
+				Failf("timed out waiting for container restart in pod=%s/%s", name, containerName)
+			}
+
+			if delay1 < 5*time.Minute {
+				continue
+			}
+		}
+
+		cappedDelay := 6 * time.Minute                          // 5mins + 1m for drift/sync loop slowness
```

Comment:
As above for this constant 6.  Rather than a comment, perhaps an expression made up of named constants?